packages/lib-webpack/src/webpack/GenerateManifestPlugin.ts

```diff
@@ -0,0 +1,24 @@
+import { pluginManifestFile } from '@openshift/dynamic-plugin-sdk';
+import type { PluginRuntimeManifest } from '@openshift/dynamic-plugin-sdk';
+import * as webpack from 'webpack';
+
+export class GenerateManifestPlugin implements webpack.WebpackPluginInstance {
+  constructor(private readonly manifest: PluginRuntimeManifest) {}
```

Comment:
I would allow passing custom `pluginManifestFile` name.

Reply:
I think that's in the `manifest` 🤔 

```
declare type PluginRuntimeMetadata = {
    name: string;
    version: string;
};
declare type PluginRuntimeManifest = PluginRuntimeMetadata & {
    extensions: Extension[];
};
```

Reply:
@andrewballantyne This is about customizing the name of plugin manifest asset, which is currently hard-coded to `plugin-manifest.json`.

I don't see a use case for that right now, but I'm not against it :smiley: